### ISSUE TITLE
Contextual menu icons could be bigger

### ISSUE THREAD

**Issue by DomeAlessi** [stride3d/stride#2429]
Hello!

I wanted to ask if it's possible to scale up a bit the size of the icons in the context menus. 
Right now they are very little and the image is very difficult to perceive even on a close monitor due the big contrast with the background.
Please refer to the attached screenshot below.

**Describe the solution you'd like**
To enhance readability, I suggest making the icons slightly larger and adjusting the contrast between the icons and the dark menu background. This change would also improve the visibility of disabled items, making the overall interface easier to navigate.

Thank you!
![image](https://github.com/user-attachments/assets/8cab2793-9a0a-4b95-b889-2b9b6ce43477)

**Comment by VaclavElias** [comment]
This is something weird! Not sure when this happened, the icons should be like this. The image is from one of the old tutorials
<img width="160" alt="image" src="https://github.com/user-attachments/assets/71a2998e-8d55-46dc-8a1a-51946de73042">

And not like this
<img width="131" alt="image" src="https://github.com/user-attachments/assets/bb6f4861-c432-4259-991f-4e9f8e0e1057">

I wonder if we made some change or Windows 11 is doing something for us 🤣

**Comment by DomeAlessi** [comment]
Well at least I am glad it's not the intended size and something is wrong somewhere :D anyway yeah I am on windows 11 and I am at 100% DPI settings on all monitors.

**Comment by Basewq** [comment]
I don't have time to have a closer look at this, but the first that comes to mind is #1807, there's a bunch of width/height value changes that may be the cause of this issue.

**Comment by DomeAlessi** [comment]
Yeah if you remove the Margin="5,5" from MenuItem ControlTemplate ContentPresenter in Stride.Core.Presentation.Wpf\Themes\ThemeSelector.xaml the icons can stretch to the max of their parent:

![image](https://github.com/user-attachments/assets/d7e643b1-b95d-47bc-ae74-f18552163b7f)

![image](https://github.com/user-attachments/assets/d3b8286b-011d-4f4b-a928-2b6a5a2f2b0b)

**Comment by DomeAlessi** [comment]
_En passant_ I see that the spacing system is kinda mixed, i see lot of multiple-of-5 margins which are not great for application scaling.
